cmd/kube-proxy/app/server_others.go

```diff
@@ -571,7 +542,7 @@ func cleanupAndExit() error {
 
 	var encounteredError bool
 	for _, ipt := range ipts {
-		encounteredError = userspace.CleanupLeftovers(ipt) || encounteredError
+
```

Comment:
possibly we should leave this for a while longer? (but if not, then you shouldn't leave a blank line behind)